### ISSUE TITLE
Enhance code quality of quarantine document generation

### ISSUE THREAD

**Issue by StefanKock** [hzi-braunschweig/SORMAS-Project#3480]
### Problem Description

Follow up to #2696. I noticed these points because of 1. that the units tests fail on my local system.

1. Running some tests on a local path with blank (' ') fail: 2x _TemplateEngineServiceTest_, 6x of 8 in _QuarantineOrderFacadeEjbTest_
2. _TemplateEngineService.getTempDir()_ is not used.
3. _TemplateEngineService.logger_ and _.configFacade_ are superfluous.
4. _TemplateEngineServiceTest_ does not need to run as BeanTest, avoids about 14s test runtime.
5. _TemplateEngineServiceTest_: XWPFWordExtractor not closed
6. _QuarantineOrderFacadeEjbTest_: XWPFWordExtractor not closed, rdcf2 not used
7. The mentioned classes do not follow the code formatting rule "Use one blank line after method definition (but usually not for one liners like getters/setters or delegation)." , see https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/CONTRIBUTING.md

### Proposed Change

- Avoid persisting the path as String but encode it like this
```
		URL file = testClass.getClassLoader().getResource("templates/" + templateFileName);
		return new FileInputStream(new File(file.toURI()));
```
- Clean up all other other points mentioned above.

**Comment by JonasCir** [comment]
@syntakker @Seal33 FYI

**Comment by syntakker** [comment]
Implemented in  #4171.
